readme.md

```diff
@@ -172,8 +173,18 @@ client.query('DESCRIBE events', (err, rows) => {
   // ..
 })
 ```
+<a name="queryAsync"></a>
+### `queryAsync(sql[, options])`
 
-I'll take a PR for Promise support.
+Same as above but promise-based.
+
+```js
+try {
+  const rows = await client.queryAsync('DESCRIBE events');
+catch(error) {
+  // ..
+}
```

Comment:
```suggestion
const rows = await client.queryAsync('DESCRIBE events')
```

Adding `try/catch` to the example is not necessary.

Reply:
Done